packages/server/package.json

```diff
@@ -200,4 +201,4 @@
   "optionalDependencies": {
     "registry-js": "1.8.0"
   }
-}
+}
```

Comment:
revert this plz. you may want to turn on the preference to "end files with a newline" in your editor